selenium_tests/course_admin/course_admin_base_test_case.py

```diff
@@ -39,4 +39,4 @@ def setUpClass(cls):
         if login_page.is_loaded():
             login_page.login_xid(cls.USERNAME, cls.PASSWORD)
         else:
-            print('(User {} already logged in to PIN)'.format(cls.USERNAME))
+            print(('(User {} already logged in to PIN)'.format(cls.USERNAME)))
```

Comment:
Unfortunately, 2to3 will keep wrapping print statements in parens. You can remove the outermost parens